### PR TITLE
Download corrected SRTM files and spelling 

### PR DESCRIPTION
**Pull request by JohannesUIBK**
Automatic filled SRTM download from viewfinderpanoramas.org should work now.

### CODE REVIEW

**Comment by fmaussion** [comment]
Thanks for this PR! 

I think there was a small misunderstanding however. The SRTM files are quite fine, it is mostly the ASTER zones (> 60° N and < 60°S) which we would like to replace with the corrected files. 

In my idea, all existing download functions will remain the same, but we will have a new `_download_dem3`[1] which could jump in in place of `_download_aster` when possible. It would allow to choose between all the possibilities. 

[1]: I don't know what the best name of this new dataset would be?

File: oggm/utils.py [hunk 1]
```diff
@@ -218,12 +245,17 @@ def _download_alternate_topo_file(fname):
...
+        'gimpdem_90m.zip',
+        'iceland.zip',
+        'northcanada.zip',
+        'svalbard.zip',
+        'AN1.zip',
+        'AN2.zip',
+        'AN3.zip',
+        'AN4.zip',
+        'AN5.zip'
     ]:
```
Comment (fmaussion) [inline oggm/utils.py]: What do AN1 to AN5 stand for?

Comment (fmaussion) [inline oggm/utils.py]: Maybe we should get rid of this test anyway, because its a maintenance burden (for example, franz joseph is missing and such)

**Comment by fmaussion** [comment]
It would also be easier to split the spell-checking and the DEM stuff into two separate PRs, if you dont mind?

File: oggm/utils.py (hunk 1 above)
Comment (JohannesUIBK) [inline oggm/utils.py]: Ah okay!
I thought it was easier to have all SRTM DEMs from one source (as they should be the same anyway!?)
I will change things the way you have suggested now over the next days.
I think a good name for the dataset was `_download_dem3_filled` if you are okay with it.

AN1 to AN5 stand for the Antarctica tiles 1-5.

Last but not least: yes, if you talk about the `fzipname` test, we should definitely check whether we can implement special conditions for special lats/lons in the `_download_dem3_filled` download function. That's what I believe would be the easiest to get the files nevertheless.

**Comment by JohannesUIBK** [comment]
Ah okay, testing failed, because I forgot to delete an "old" line in test_utils.py....I will do it or fall back to the original version and include your suggested changes.

Okay, we can split the PR as the spelling is hopefully correct and you can accept this PR already.

**Comment by fmaussion** [comment]
Yes, lets do the spell check first (because it might also fail, for different reasons)

File: oggm/utils.py (hunk 1 above)
Comment (fmaussion) [inline oggm/utils.py]: Yes, maybe it will replace SRTM also, but this requires careful checking. And it's always good to keep all possible options open

**Pull request closed without merging**